mscore/dragelement.cpp

```diff
@@ -75,10 +75,14 @@ void ScoreView::doDragElement(QMouseEvent* ev)
                   mscore->play(e);
                   _score->setPlayNote(false);
                   }
-            QLineF anchor = e->dragAnchor();
+            QVector<QLineF> anchorLines = e->dragAnchorLines();
+            const QPointF pageOffset(e->findAncestor(ElementType::PAGE)->pos());
```

Comment:
I think this might be where we are getting the extra page offset that is causing https://musescore.org/en/node/306096.  If I comment out the translate that happens below, it fixes the case at hand, but I'm not totally sure it doesn't break something else.